### PR TITLE
Description of `executeRelayCall(...)` updated

### PR DESCRIPTION
**Pull request by b00ste**

### CODE REVIEW

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 1]
```diff
@@ -128,6 +128,48 @@ function executeRelayCall(
...
+```
+4. After that you can sign the encodedMessageHash and voila, you have the signature ready.
+5. To execute the `payload` you would have to do the following:
+```solidity
+ILSP6KeyManager(keyManagerAddress).executeRelayCall(
+    <The signature that you got from step 4.>,
+    nonce, // We got it in step 1.
+    payload //We got it in step 1.
+);
+```
```
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:171]: I think it looks cleaner in the code if you add spaces before and after the ``` for the code block

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 2]
```diff
@@ -128,6 +128,48 @@ function executeRelayCall(
 
 Allows anybody to execute a payload on the **LSP0ERC725Account** contract, if they have a signed message from an executor.
 
+To have a valid signature you must do the following:
+1. You have to gather 4 things:
```
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:132]:
```suggestion
To have a valid signature you must do the following:

1. You have to gather 4 things:
```

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 3]
```diff
@@ -128,6 +128,48 @@ function executeRelayCall(
...
+    ...[<The parameters that will be passed to the methods>]
+);
+
+// The chain id of the blockchain where the `payload` will be executed
+uint256 chainId = block.chainid;
+or
+uint256 chainId = <The chain id of the blockchain where you will interact with the key manager>;
+
+// The address of the key manager (the smart contract where the `payload` will be executed)
+address keyManagerAddress = '0x...'
```
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:145]:
```suggestion
address keyManagerAddress = '0x...';
```

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 4]
```diff
@@ -128,6 +128,56 @@ function executeRelayCall(
...
+```solidity
+bytes memory payload = abi.encodeWithSignature(
+    "<Signature of the method that will be executed, e.g. 'setData(bytes32[],bytes[])'>",
+    ...[<The parameters that will be passed to the methods>]
+);
+
+// The chain id of the blockchain where the `payload` will be executed
+uint256 chainId = block.chainid;
+or
+uint256 chainId = <The chain id of the blockchain where you will interact with the key manager>;
```
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:144]:
```suggestion
uint256 chainId = block.chainid; // or <The chain id of the blockchain where you will interact with the key manager>
```

**Review by Hugoo — APPROVED** [review]
Let's wait for Jean and/or Yamen review before merging.

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 5]
```diff
@@ -128,6 +128,56 @@ function executeRelayCall(
 
 Allows anybody to execute a payload on the **LSP0ERC725Account** contract, if they have a signed message from an executor.
 
+To have a valid signature you must do the following:
+
+1. You have to gather 4 things:
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp6-key-manager.md:133]:
```suggestion
1. gather 4 things:
```

File: docs/standards/smart-contracts/lsp6-key-manager.md [hunk 6]
```diff
@@ -128,6 +128,56 @@ function executeRelayCall(
...
+or
+uint256 chainId = <The chain id of the blockchain where you will interact with the key manager>;
+
+// The address of the key manager (the smart contract where the `payload` will be executed)
+address keyManagerAddress = '0x...';
+
+uint256 nonce = ILSP6KeyManager(keyManagerAddress).getNonce(...);
+```
+
+2. After you gathered those 4 thing you must encode them using `abi.encodePacked(...)`:
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp6-key-manager.md:152]:
```suggestion
2. Once you have gathered these 4 informations, you must encode them using `abi.encodePacked(...)`:
```

**Review by YamenMerhi — APPROVED** [review]

**Pull request merged**